packages/rest-api-client/src/client/types/app/adminNotes.ts

```diff
@@ -0,0 +1,11 @@
+import type { AppID, Revision } from "..";
+
+export type AdminNotes = {
+  content: string;
+  includeInTemplateAndDuplicates: boolean;
+};
+
+export type AdminNoteForParameter = Partial<AdminNotes> & {
```

Comment:
This type is better to be pluralized.

```suggestion
export type AdminNotesForParameter = Partial<AdminNotes> & {
```

Reply:
[9a91117](https://github.com/kintone/js-sdk/pull/2849/commits/9a91117e943858effca9038f91ef46d61e541efd)